#!/bin/sh -e
## 20_user_agent_tag.dpatch.dpatch by Andres Salomon <dilinger@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP:  Make the Browser string recognizeable as Debian.

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -urN a/module/embed/gecko/kz-gecko-single.cpp b/module/embed/gecko/kz-gecko-single.cpp
--- a/module/embed/gecko/kz-gecko-single.cpp	2008-02-20 19:31:10.000000000 +0000
+++ b/module/embed/gecko/kz-gecko-single.cpp	2008-02-20 19:31:44.000000000 +0000
@@ -323,7 +323,7 @@ set_user_agent (KzProfile *profile)
 	{
 		mozilla_prefs_clear("general.useragent.override");
 		mozilla_prefs_set_string("general.useragent.extra.kazehakase",
-					 "Kazehakase/" VERSION);
+					 "Kazehakase/" VERSION " Debian/" DEBIAN_VERSION);
 	}
 }
 
